datahub-web-react/src/app/glossary/GlossaryBrowser/GlossaryBrowser.tsx

```diff
@@ -59,6 +63,17 @@ function GlossaryBrowser(props: Props) {
         }
     }, [refreshBrowser, refetchNodes, refetchTerms]);
 
+    useEffect(() => {
+        if (updatedUrns.includes(ROOT_NODES)) {
```

Comment:
Special case handling for rootNodes and rootTerms? (Where there is no parent) Got it. Might be worth adding a comment somewhere

Reply:
definitely agreed. a few comments around this change in general wouldn't hurt.